src/core/moves/proposal/tree/SubtreeSwapProposal.cpp

```diff
@@ -97,12 +97,22 @@ bool SubtreeSwapProposal::isDescendant(const TopologyNode &n, const TopologyNode
  */
 double SubtreeSwapProposal::doProposal( void )
 {
+    // reset flag
+    failed = false;
     
     // Get random number generator
     RandomNumberGenerator* rng     = GLOBAL_RNG;
     
     Tree& tau = tree->getValue();
     
+    size_t num_tips = tau.getNumberOfTips();
+    size_t num_root_children = tau.getRoot().getNumberOfChildren();
+    if ( num_tips <= num_root_children)
```

Comment:
Why can't you test num_tips <= 4? Also, 4 should be fine. So maybe 3?

Reply:
Please see the response above to the previous comment on the same code.